knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -602,6 +602,14 @@ func configureEventingKafka(spec serverlessoperatorv1alpha1.KnativeKafkaSpec) mf
 			return scheme.Scheme.Convert(deployment, u, nil)
 		}
 
+		if u.GetKind() == "ConfigMap" && u.GetName() == "kafka-config-logging" {
+			log.Info("Found ConfigMap kafka-config-logging, updating it with values from spec")
+			xmlTemplate := "    <configuration>\n      <appender name=\"jsonConsoleAppender\" class=\"ch.qos.logback.core.ConsoleAppender\">\n        <encoder class=\"net.logstash.logback.encoder.LogstashEncoder\"/>\n      </appender>\n      <root level=\"" + spec.Logging.Level + "\">\n        <appender-ref ref=\"jsonConsoleAppender\"/>\n      </root>\n    </configuration>"
```

Comment:
On purpose I am avoiding proper xml usage, to keep this _simple_ instead of using `encoding/xml` and friends...

Reply:
Nit picking:

We can actually have a template file and use https://pkg.go.dev/embed.

I am ok with this too though.

But at least use raw string literals?

Reply:
I changed to raw strings. Let's not use the template engine.

I also extracted the "xml rendering" fu to some (testable) function, to make the actual code of the transformers less verbose.

thanks @aliok